diff --git a/node_modules/@pandacss/core/dist/index.js b/node_modules/@pandacss/core/dist/index.js
index cde18a1..eb69b62 100644
--- a/node_modules/@pandacss/core/dist/index.js
+++ b/node_modules/@pandacss/core/dist/index.js
@@ -370,13 +370,12 @@ var Conditions = class {
     containerNames.unshift("");
     containerNames.forEach((name) => {
       Object.entries(containerSizes).forEach(([size, value]) => {
-        const _value = (0, import_shared2.toEm)(value) ?? value;
         containers[`@${name}/${size}`] = {
           type: "at-rule",
           name: "container",
           raw: name,
-          value: _value,
-          rawValue: `@container ${name} (min-width: ${_value})`,
+          value: value,
+          rawValue: `@container ${name} (min-width: ${value})`,
           params: `${name} ${value}`
         };
       });
